src/index.d.ts

```diff
@@ -0,0 +1 @@
+// #
```

Comment:
do we need this?

Reply:
I was getting an error and it was suggested to have a file named index.d.ts. So I added that. I will look into the issue properly.